### PR TITLE
Feat/pritipsingh/410

### LINKED ISSUE #410: [FEAT] - New Warning System that works with Mongo
### What feature would you like to see?

New Warning System that works with Mongo

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by pritipsingh**
# Pull Request

## Description

Added a basic warning system. Included a mongo schema and wrote 2 commands: 
1. Warn.js - this command warns the mentioned user.
2. warning.js - checks the number of warnings of the mentioned user

## Related Issue

Closes #410 

## Version
Did you remember to update the version number in the package.json file?

- [ ] For a nmajor restructuring, or large changes please bump the the major version number. (ex: 1.x.x)
- [x] For a minor change, bump the minor version number. (ex: x.1.x)
- [ ] For a bug fix, bump the patch version number. (ex: x.x.1)

## Documentation

For _any_ version updates, please verify if the [documentation page](https://docs.beccalyria.com?utm_source=github&utm_medium=pr-template) needs an update. If it does, please [create an issue there](https://github.com/BeccaLyria/discord-documentation/issues/new?assignees=nhcarrigan&labels=%F0%9F%9A%A6+status%3A+awaiting+triage&template=update.md&title=%5BUPDATE%5D) to ensure it is not forgotten.

- [ ] My contribution does NOT require a documentation update.
- [ ] My contribution DOES require a documentation update.

### CODE REVIEW

**Comment by j-dogcoder** [comment]
Hey @pritipsingh, please bump the minor version number, and edit the PR description with the issue this PR closes.

File: package.json [hunk 1]
```diff
@@ -2,10 +2,10 @@
   "name": "heptagram",
   "version": "2.0.0",
```
Comment (j-dogcoder) [inline package.json:3]: Please bump version

Comment (pritipsingh) [reply package.json]: sure thing!

File: src/commands/warn.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  //     iconURL: `${client.config.cdn.sqlogo}`,
+  //   });
+
+//   try {
+//     await message.guild.members.warn(user, { reason });
+//   } catch (error) {
+//     return message.reply({ embeds: [errorembed] });
+//   }
+
+//   return message.reply({ embeds: [warnembed] });
```
Comment (j-dogcoder) [inline src/commands/warn.js:79]: Why is this commented out?

File: src/commands/warning.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  //     iconURL: `${client.config.cdn.sqlogo}`,
+  //   });
+
+//   try {
+//     await message.guild.members.warn(user, { reason });
+//   } catch (error) {
+//     return message.reply({ embeds: [errorembed] });
+//   }
+
+//   return message.reply({ embeds: [warnembed] });
```
Comment (j-dogcoder) [inline src/commands/warning.js:66]: Why is this commented out?

**Review by jaspermayone — CHANGES_REQUESTED** [review]

**Comment by j-dogcoder** [comment]
Thanks so much for being patient @pritipsingh! I left you some comments.

File: src/commands/warning.js (hunk 3 above)
Comment (pritipsingh) [reply src/commands/warning.js:66]: I was a bit confused if I should add the error message or not, so instead waited for your comment. **I'll go ahead and add it?**

Comment (j-dogcoder) [reply src/commands/warning.js:66]: Sounds good! Yes, please add it.

**Review by jaspermayone — APPROVED** [review]

**Comment by j-dogcoder** [comment]
### 🎉 Congratulations @pritipsingh! 🎉 

Congratulations on making your first pull request to our project. We have reviewed your contributions and are happy to accept them.

We hope you enjoyed the contribution process, and encourage you to watch for issues labeled `help wanted` for additional opportunities to contribute.

**Discord Server:**
_We would love it if you joined our community discord server! Your contribution to the project make you eligible for special roles & perks._

**Comment by pritipsingh** [comment]
Hey, @j-dogcoder I am already on the discord server. My id is Priti#2368 . Looking forward to contributing more post my exams.

**Pull request merged**